build.gradle

```diff
@@ -102,18 +105,35 @@ application {
     // Take the application out once we're running via Cassandra
     mainClassName = "org.apache.cassandra.sidecar.CassandraSidecarDaemon"
     applicationName = 'cassandra-sidecar'
-    // Config file location should be in file:/// format for local files,
-    def confFile = "file:" + File.separator + File.separator + "APP_HOME_TO_REPLACE/conf/sidecar.yaml"
-    applicationDefaultJvmArgs = ["-Dsidecar.logdir=./logs",
-                                 "-Dsidecar.config=" + confFile,
-                                 "-Dlogback.configurationFile=./conf/logback.xml",
-                                 "-Dvertx.logger-delegate-factory-class-name=io.vertx.core.logging.SLF4JLogDelegateFactory",
-                                 "-javaagent:APP_HOME_TO_REPLACE/agents/jolokia-jvm-1.6.0-agent.jar=port=7777,host=localhost"]
+    applicationDefaultJvmArgs = ["PLACEHOLDER"]
```

Comment:
Just for my education, what is this placeholder here and on line 134?

Reply:
When the `startScripts` Gradle task runs, it will create a variable in the start script called `DEFAULT_JVM_OPTIONS` which is set to the values in `applicaitonDefaultJvmArgs`. This variable is purposefully created in a way such that the environment variables that we want to embed into the `DEFAULT_JVM_OPTIONS` will not be evaluated.

The way around this is to generate `DEFAULT_JVM_OPTIONS='PLACEHOLDER'` in the start script, then modify that value after the start script is generated and replace `'PLACEHOLDER'` with the actual default JVM options.